1-python-wrapper/wrapper/main.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask, request
+from model.model import load_most_recent_model
+import pandas as pd
+import logging
+
+
+logger = logging.getLogger(__name__)
+app = Flask(__name__)
+
+@app.route('/predict', methods=["GET", "POST"])
+def predict():
+    if request.method == "POST":
+        print(schema)
+        input = request.json
+        data = pd.DataFrame(input, index=[0])
+        return {"classification": str(model.predict(data)[0])}
+    else:
+        return "Please provide post request according to schema"
+
+
+@app.route('/schema', methods=["GET"])
+def schema():
+    return schema
```

Comment:
Where is this variable defined?

Reply:
When starting the flask app, the most recent model is loaded, together with schema and metadata. I am not aware of a better method, any ideas?

Reply:
Nope, this was to be expected: at startup the schema needs to get loaded from file, as well as the model needs to get initialized (possibly also loading a file from disk).